components/EntireScheme.svelte

```diff
@@ -63,15 +65,37 @@
     document.body.removeChild(element);
   }
 
+  function checkOrigin(geojson) {
+    if (geojson.hasOwnProperty("origin")) {
+      let origin = geojson.origin;
+      return origin.startsWith("atip-");
+    } else if (
+      geojson.hasOwnProperty("authority") &&
+      !geojson.hasOwnProperty("origin")
+    ) {
+      return true;
+    } else {
+      return false;
```

Comment:
We can simplify this in the else block; we know origin isn't set

```suggestion
    } else {
      return geojson.hasOwnProperty("authority");
    }
```